pages/scripts/index.js

```diff
@@ -333,4 +338,24 @@ document.addEventListener('keydown', (event) => {
   if (event.keyCode >= 65 && event.keyCode <= 90) {
     document.querySelector('cr-search-control').inputElement.focus();
   }
-})
+  // Escape key
+  if (event.keyCode === 27 && aside.classList.contains('shown')) {
+    aside.classList.remove('shown');
+  }
+});
+
+menuNavigationLink.addEventListener('click', (event) => {
+  // Don't trigger the click event on the main section
+  event.stopPropagation();
+  aside.addEventListener('transitionend', () => {
+    // Move focus into close button of drawer
+    asideCloseButton.focus();
+  });
```

Comment:
```suggestion
  }, {once: true});
```

otherwise this is bound on every click
![image](https://user-images.githubusercontent.com/39191/80434481-b9bbc880-88ae-11ea-81a9-70b2f4ce8a52.png)

Reply:
Good catch!